dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -5125,6 +5135,8 @@ definitions:
         type: "integer"
         format: "int64"
         description: "Implementation specific ID for the container in this web service"
+      checker:
```

Comment:
Can you provide a description of what `checker` is?  Especially when compared to isChecker and check_id

Reply:
Weird I have no idea where that came from, there should just be is_checker and checker_id. I wonder if it was an automatic thing added for the is_checker.

Reply:
Oops found it. It is from intellij autogen code.